### DISCUSSION TITLE
How To View Specific Country

### DISCUSSION THREAD

**Discussion by ra-heang** [inocan-group/vue3-google-map#210] [Q&A] [answered]
Example: ['France','United State',.....]
Please, demo me. Thanks for more.

**Comment by WalidBouhsin9** [comment, marked as answer]
use the `:region="'country-code'"` attribute like this` :region="'fr'" ` 
Example :
`<GoogleMap id="map" :region="'ma'" api-key="your-api-key"> </GoogleMap>`

**Reply by ra-heang** [reply]
Very thanks
